examples/run_classifier.py

```diff
@@ -626,6 +626,10 @@ def main():
                   'global_step': global_step,
                   'loss': tr_loss/nb_tr_steps}
 
+        model_to_save = model.module if hasattr(model, 'module') else model
+        raise NotImplementedError # TODO add save of the configuration file and vocabulary file also ?
+        output_model_file = os.path.join(args.output_dir, "pytorch_model.bin")
+        torch.save(model_to_save, output_model_file)
```

Comment:
Hi!
Just flagging that the recommended practice here is to save a state_dict as per https://pytorch.org/docs/stable/notes/serialization.html